src/features/projectsV2/ProjectListControls/index.tsx

```diff
@@ -78,7 +80,15 @@ const ProjectListControls = ({
               })}
             </div>
           ) : (
-            <ProjectListTabLargeScreen {...projectListTabProps} />
+            <>
+              {isSalesforceTenant ? (
+                <h2 className={styles.salesforceHeaderText}>
+                  {tCommon('stopTalkingStartPlanting')}
+                </h2>
+              ) : (
+                <ProjectListTabLargeScreen {...projectListTabProps} />
+              )}
+            </>
           )}
           <SearchAndFilter {...searchAndFilterProps} />
         </div>
```

Comment:
The logic here is a little cluttered - too many levels of nesting in the ternary statement. Simplify this.